src/master-playlist-controller.js

```diff
@@ -698,7 +698,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.masterPlaylistLoader_.media(media);
 
       this.mainSegmentLoader_.resetLoader();
-      if (this.audiosegmentloader_) {
```

Comment:
not only was this not checking for the playlist loader, it's not cased properly for the segment loader lol